### DISCUSSION TITLE
Homematic IP Geräte reagieren nicht mehr

### DISCUSSION THREAD

**Discussion by FlorianLoch** [OpenCCU/OpenCCU#2903] [🇩🇪 Fragen&Antworten] [answered]
Hallo,
auf einem Raspberry Pi 5 betreibe ich in Docker RaspberryMatic (3.79.6.20241031). Verbunden sind einige Thermostate und Fenstersensoren. Das hat bisher wunderbar funktioniert - seit wenigen Tagen scheint die Kommunikation jedoch gänzlich gestört zu sein.

Die Sensoren/Aktoren können noch untereinander kommunizieren, die direkten Verbindungen zwischen Fenstersensoren und Thermostaten funktionieren weiterhin. Allerdings aktualisiert sich der Status in RaspberryMatic nicht mehr und es ist auch keine Steuerung von dort aus mehr möglich

> Ventilantrieb Status:
Ventilstatus unbekannt

Duty Cycle und Carrier Sense bewegen sich < 5 %. Entferne ich den HmIP-RFUSB, so "merkt" das System das. 

Zum Test habe ich einen der Fenstersensoren zurückgesetzt und versucht ihn erneut anzulernen - jedoch ohne Erfolg.

Ideen, was man hier machen kann? ;)

[homematic-raspi-2024-11-06.log](https://github.com/user-attachments/files/17648959/homematic-raspi-2024-11-06.1.log)

**Comment by FlorianLoch** [comment]
Ich habe nun nochmal mit einem neuen Setup mein Glück versucht - RaspberryMatic erkennt den RFUSB, findet dann jedoch keine Sensoren/Aktoren bzw. diese lassen sich nicht anlernen.

Ich habe den Stick bezüglich kalter Lötstellen durchgemessen - scheint alles wunderbar zu sein.

Sehr frustrierend - zumal mir so langsam die Ideen ausgehen. Bin für jede Anregung dankbar ;)

**Comment by FlorianLoch** [comment]
Eine Sache habe ich gerade noch herausgefunden: den genauen Zeitpunkt, ab dem RaspberryMatic keine Werte mehr geliefert hat. So früh am Morgen, dass ich sicher sagen kann, dass ich da nichts am System geändert hate. 🤔

**Comment by FlorianLoch** [comment]
Nach ein paar weiteren Versuchen habe ich nun wenigstens noch einen "besseren" Log erzeugt bekommen. Wenn ich das richtig interpretiere, dann bekommt RaspberryMatic keine Nachrichten über den Stick versendet? Einen Grund bzw. eine Lösung kann ich daraus allerdings nicht ableiten. :/ 

[Uploading homematic-raspi-2024-11-06 (3).log…]()

**Comment by Baxxy13** [comment]
Das "bessere Log" ist nicht herunterladbar.
Außerdem fehlt die Info welche Geräte nicht mehr ansprechbar sind.
- HM
- HmIP
- Alle

Dein Log ist "too much", also strukturiert den Log-Level für das jeweilige Problem hochsetzen.
Die Logikschicht schließen wir mal aus---> Loglevel "nur Fehler"
HM-Geräte --> "Homematic Funk" Loglevel "alles loggen"
HmIP-Geräte --> "Homematic IP" Loglevel "TRACE"

**Comment by FlorianLoch** [comment]
Oh, mein Fehler. Habe die Log-Level entsprechend gesetzt und, um das Log ein wenig aufzufüllen, versucht die Thermostate manuell via Web UI einzustellen.

Ich habe nur HmIP Geräte, diese sind jedoch alle betroffen.

Danke vorab :)

[homematic-raspi-2024-11-06 (4).log](https://github.com/user-attachments/files/17652160/homematic-raspi-2024-11-06.4.log)

**Reply by Baxxy13** [reply]
Ich denke du hast einen Funkstörer irgendwo.
Vielleicht nicht dauerhaft aber zumindest sporadisch.
Du schreibst zwar...
> Duty Cycle und Carrier Sense bewegen sich < 5 %

... dein Log sagt aber zeitweise was ganz anderes:
```
2024-11-06 20:33:59,971 de.eq3.cbcs.lib.hmiptrxcommadapter.internal.TRXAdapter DEBUG [RXTXPortMonitor(/dev/mmd_hmip)] AP 3014F711A000041F2995EAEB: carrier sense event: value = 10.0%, value is rising
2024-11-06 20:34:00,668 de.eq3.cbcs.lib.hmiptrxcommadapter.internal.TRXAdapter DEBUG [RXTXPortMonitor(/dev/mmd_hmip)] AP 3014F711A000041F2995EAEB: carrier sense event: value = 20.0%, value is rising
2024-11-06 20:34:01,578 de.eq3.cbcs.lib.hmiptrxcommadapter.internal.TRXAdapter DEBUG [RXTXPortMonitor(/dev/mmd_hmip)] AP 3014F711A000041F2995EAEB: carrier sense event: value = 30.0%, value is rising
....
2024-11-06 20:34:09,700 de.eq3.cbcs.lib.hmiptrxcommadapter.internal.TRXAdapter DEBUG [RXTXPortMonitor(/dev/mmd_hmip)] AP 3014F711A000041F2995EAEB: carrier sense event: value = 80.0%, value is rising
2024-11-06 20:34:11,448 de.eq3.cbcs.lib.hmiptrxcommadapter.internal.TRXAdapter DEBUG [RXTXPortMonitor(/dev/mmd_hmip)] AP 3014F711A000041F2995EAEB: carrier sense event: value = 70.0%, value is falling
```

**Reply by FlorianLoch** [reply]
Daran habe ich auch schon gedacht, auch wenn ich diese hohen Werte tatsächlich im UI selbst nicht gesehen habe. Nützlich zu wissen, dass sich das im Log findet. Das spricht ja auch auf jeden Fall dafür, dass der Stick an sich (weiterhin) zu funktionieren scheint und das Problem woanders zu suchen ist. Das wäre ja schonmal erfreulich.

Ich habe vorhin bereits alle möglichen anderen Geräten abgezogen, allerdings ohne Erfolg. Ich werde nochmal scharf nachdenken, was sich seit gestern Morgen verändert haben könnte - bis jetzt ist mir das nämlich noch überhaupt nicht klar :/ 

Jetzt sehe ich auch folgende Meldung in den Alarmen: `CarrierSense 100%`. Sehe ich allerdings zum ersten Mal.

Danke!

**Reply by FlorianLoch** [reply]
Nachdem RaspberryMatic über mehrere Minuten eine Auslastung der Frequenz von 100 % vermeldet hat, habe ich nun nochmal gesucht und überlegt. Hat keine Veränderung gebracht. Nach einem Neustart ist der Wert allerdings wieder zwischen 0 - 5 %. Auch im Log findet sich nun nichts mehr. 

Kommunikation ist aber weiterhin gestört, Thermostate melden keinen Ventilstatus und eine noch ausstehende Config-Nachricht wurde immer noch nicht übertragen.

Scheint für mich nun doch ein wenig so, als wäre das ein Hardware-Defekt? Auch wenn es mir alles ein wenig merkwürdig vorkommt :/

**Comment by FlorianLoch** [comment, marked as answer]
Nachdem ich keine Quelle für mögliche Funktstörungen finden konnte und es mir außerdem merkwürdig vorkam, dass der HM-RFUSB bei anhaltender Meldung von 100% Frequenzbelegung nach einem Neustart eine solche plötzlich nicht mehr signalisierte, habe ich den Stick nun einfach ersetzt. 

Da der HM-RFUSB im Moment schwer zu bekommen ist (Lieferzeit als Bausatz bei ELV > 30 Wochen (!)) habe ich alternativ den HM-RFUSB-TK (also die Telekom-Edition) besorgt. Nun funktioniert wieder alles. Ich gehe also davon aus, dass es tatsächlich an einem Hardwaredefekt gelegen haben muss, anders kann ich es mir nicht erklären.

Danke dennoch für die Hilfe.
